docs/upgrading-to-70.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[upgrading-to-70]]
+=== Upgrading to APM Server v7.0
+
+Before upgrading to APM Server v7.0,
+there are some {apm-overview-ref-v}/breaking-7.0.0-beta1.html[breaking changes]
+in the APM Server and the APM UI that you should be aware of.
+
+[[upgrade-steps-70]]
+==== Upgrade Steps
+
+Check the https://www.elastic.co/support/matrix#matrix_compatibility[Product Compatibility matrix]
+to determine if you need to upgrade Elasticsearch and Kibana. 
+
+. Upgrade {ref}/setup-upgrade.html[Elasticsearch]
+. Upgrade {kibana-ref}/upgrade.html[Kibana]
+. Ensure all of your APM agents are upgraded to a version that supports APM Server >= 6.5.
+The {apm-overview-ref-v}/agent-server-compatibility.html[agent/server compatibility matrix]
+will help determine compatibility.
+. Upgrade APM Server. (See below if upgrading from an RPM or Debian install)
```

Comment:
annoying nitpick: RPM is a package format, Debian is a Linux distribution - i'd suggest Deb instead of Debian as package formats work for several distributions 

(same bellow)